app/src/ui/dropdown-select-button.tsx

```diff
@@ -145,6 +200,16 @@ export class DropdownSelectButton<
     )
   }
 
+  private renderOption = (o: IDropdownSelectButtonOption<T>) => {
+    return (
+      <Button key={o.value} onClick={this.onSelectionChange(o)}>
```

Comment:
Buttons might feel weird here as list item, but they naturally give us the accessibility features of being a selectable input without having to implement custom logic to focus and put keypress/onclicks on li or divs which then means we have to ignore linter rules. Also, it is what dotcom does as well.